test/images/nvidia-cuda/Makefile

```diff
@@ -0,0 +1,27 @@
+# Copyright 2017 The Kubernetes Authors.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+.PHONY: all push build
+
+TAG ?= v0.1
+
+REGISTRY ?= gcr.io/google-containers
+IMAGE = $(REGISTRY)/cuda-vector-add
+
+build:
+	docker build --pull -t $(IMAGE):$(TAG) .
```

Comment:
Good point. I'd like to do it later since I don't know the multi-arch story for CUDA yet.